code/modules/hydroponics/plant_genes.dm

```diff
@@ -250,14 +250,14 @@
 /datum/plant_gene/trait/cell_charge/on_slip(obj/item/reagent_containers/food/snacks/grown/G, mob/living/carbon/C)
 	var/power = G.seed.potency*rate
 	if(prob(power))
-		C.electrocute_act(round(power), G, 1, SHOCK_NOGLOVES)
+		C.electrocute_act(round(power), G, 1, 1)
 
 /datum/plant_gene/trait/cell_charge/on_squash(obj/item/reagent_containers/food/snacks/grown/G, atom/target)
 	if(iscarbon(target))
 		var/mob/living/carbon/C = target
 		var/power = G.seed.potency*rate
 		if(prob(power))
-			C.electrocute_act(round(power), G, 1, SHOCK_NOGLOVES)
+			C.electrocute_act(round(power), G, 1, 1)
 
```

Comment:
why the define replacement?

Reply:
i have no idea where this change came from, this is my first time using git so i must have pulled the wrong file from somewhere? i fixed it in the below commit hopefully